vault/logical_system.go

```diff
@@ -2563,14 +2549,22 @@ func (b *SystemBackend) handlePoliciesList(policyType PolicyType) framework.Oper
 		case PolicyTypeACL:
 			// Add the special "root" policy if not egp
 			policies = append(policies, "root")
-			return logical.ListResponse(policies), nil
+			resp := logical.ListResponse(policies)
+
+			fmt.Println(req.Path)
+			// If the request is from sys/policy/ we handle backwards compatibility
+			if strings.HasPrefix(req.Path, "policy") {
+				resp.Data["policies"] = resp.Data["keys"]
```

Comment:
Isn't this missing the code to add the root policy?

Reply:
It's in https://github.com/hashicorp/vault/pull/4677/files/af11a3a72a3505362acdbce719470109d9802fc2#diff-a145099c8e3d917858619dcfb2ae09b9R2551

Reply:
oh nm I missed it